clients/filesystem-hadoop3/src/test/java/org/apache/gravitino/filesystem/hadoop/integration/test/GravitinoVirtualFileSystemGCSIT.java

```diff
@@ -88,6 +88,7 @@ public void startUp() throws Exception {
     conf.set("fs.gvfs.impl.disable.cache", "true");
```

Comment:
could you create a seperate IT for testing HDFS, then the cloud IT will not start HDFS container to reduce test time

Reply:
The cloud IT will not start Hive and HDFS, please check it